packages/mobile_app/lib/components/yesterday_poll.dart

```diff
@@ -31,7 +31,7 @@ Padding yesterdayPollSimple(
           SizedBox(height: 16),
           MagicPollPicker(
             onTap: () {
-              userPollBloc.repo.todayPoll.pollSetAll(mood.wrapped);
+              userPollBloc.current.pollSetAll(mood.wrapped);
```

Comment:
I forgot to change it long ago. It actually does nothing, but more error-proof and less-code.